src/pages/fundamentals/eutxo.mdx

```diff
@@ -393,13 +392,13 @@ The redeemer, on the other hand, is another piece of data that is also provided
 with the transaction for any script execution. Notice that the datum and
 redeemer intervene at two distinct moments. A datum is set when the output is
 created (i.e. when the post-it note is hung on the wall, it is part of the
-note). Whereas the redeemer is provided only when spending the output (i.e.
-with the form handed over to the employee).
+note). In contrast,  the redeemer is provided only when spending the output (i.e.
+provided along with the form as it is handed over to the employee).
 
 ### Analogy
 
-Another way to look at scripts, datums and redeemers is to think of them as a
-parameterised mathematical function.
+Another way to look at scripts, datums and redeemers is to think of them as forming a
+parameterised mathematical functions.
```

Comment:
This one doesn't sound right? Shouldn't this stay singular?

Reply:
Yes, my mistake: the "a" was supposed to be deleted as in:
"Another way to look at scripts, datums and redeemers is to think of them as forming parameterised mathematical functions."
Ironically this mistaken mixing of plural/singular was what I intended to solve lol. Fixed